releasenotes/notes/solovay-kitaev-transpiler-pass-bc256c2f3aac28c6.yaml

```diff
@@ -0,0 +1,24 @@
+---
+features:
+  - |
```

Comment:
We should probably have feature notes about the transpiler pass and the unitary synthesis plugin in addition to this.

Reply:
done in [902dbda](https://github.com/Qiskit/qiskit-terra/pull/5657/commits/902dbdae67357f4eb3897aa936d574862a7e8ed8)